.github/workflows/release-bump-version.yml

```diff
@@ -1,6 +1,8 @@
 name: Release - Bump Version
 
-on:
+on:  # yamllint disable-line rule:truthy
+  schedule:
+    - cron: '0 0 1 * *'
```

Comment:
we usually add the crontab link as a comment so it's easy to look up https://crontab.guru/#0_0_1_*_*